testing/README.md

```diff
@@ -0,0 +1,75 @@
+### E2E tests
+
+#### Dependencies
+
+[oblt-cli](https://studious-disco-k66oojq.pages.github.io)
+
+### Try it locally
+
+0. Setup vault
+Export the environment variable `VAULT_TOKEN` or have the `~/.vault-token` file created.
+For further details about how to configure Vault check [the Vault documentation](https://github.com/elastic/infra/tree/master/docs/vault)
```

Comment:
FYI this is a private repository link going into a public repository. Also is this actually necessary? What is it used for? It doesn't appear in any of the oblt-cli documentation.

Reply:
It appears the oblt-cli releases and repositories as also all private.

Likely this documentation should move to the private ingest repository since it is going to link to several private repositories and not be usable by anyone that isn't an Elastic employee.

Reply:
Created a PR to the ingest repo

Reply:
> Created a PR to the ingest repo

Given the PR in the ingest repo, I think this file should be removed from this PR so we don't end up with instructions in two repos.